.storybook/cookie-policy.ts

```diff
@@ -0,0 +1,49 @@
+const COOKIE_ACCEPT_GDPR_NAME = 'GDPR';
+const COOKIE_ACCEPT_VALUE = 'accepted';
+
+function setGDPRCookie(expirationInDays = 365) {
```

Comment:
follow coding styles
```suggestion
const setGdprCookie = (expirationInDays = 365) => {
```

Reply:
why this?

Reply:
I mean, no problem at all, but "follow coding style" is not so good to get the grasp of your mindset :D

Reply:
there should be a good reason to not follow the coding style of the rest of the repo